File: TeamCode/Proto/src/main/java/org/murraybridgebunyips/proto/debug/Noop.java
package org.murraybridgebunyips.proto.debug;

import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

import org.murraybridgebunyips.bunyipslib.BunyipsOpMode;
import org.murraybridgebunyips.proto.Proto;

/**
 * No-op to init hardware and print timer status.
 */
@TeleOp
public class Noop extends BunyipsOpMode {
    private final Proto robot = new Proto();

    @Override
    protected void onInit() {
        robot.init();
    }

    @Override
    protected void activeLoop() {
        telemetry.add(timer);
    }
}

File: TeamCode/Proto/src/main/java/au/edu/sa/mbhs/studentrobotics/ftc15215/proto/debug/Noop.kt
package au.edu.sa.mbhs.studentrobotics.ftc15215.proto.debug

import au.edu.sa.mbhs.studentrobotics.bunyipslib.BunyipsOpMode
import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import au.edu.sa.mbhs.studentrobotics.ftc15215.proto.Proto

/**
 * No-op to init hardware and print timer status.
 */
@TeleOp(name = "No-op", group = "a")
class Noop : BunyipsOpMode() {
    private val robot = Proto()

    override fun onInit() {
        robot.init()
    }

    override fun activeLoop() {
        telemetry.add(timer)
    }
}
